C-Plus-Plus/other/lcm.cpp

```diff
@@ -0,0 +1,55 @@
+//Using GCD, find LCM of two number.
+
+#include <iostream>
+using namespace std;
+
+// Gcd recursive Function
+long long gcd(long long int x, long long int y)
+{
+  if (y == 0)
+    return x;
+    //Rcursive function
+  return gcd(y, x % y);
+}
+
+// Lcm Function
+long long lcm(int a, int b)
+{
+	//return Lcm of two number
+    return (a / gcd(a, b)) * b;
+}
+
+//main function
+int main()
+{
+	//Input 2 numbers for Lcm
+    int n1, n2, max;
```

Comment:
```suggestion
    int n1, n2;
```
We don't need any extra variable here.